utils/src/test/scala/cromwell/pipeline/utils/TestContainersUtils.scala

```diff
@@ -27,16 +25,19 @@ object TestContainersUtils {
   private lazy val mongoConfig = ConfigFactory.load().getConfig("database.mongo")
   private def mongoPort: Int = mongoConfig.getInt("port")
 
-  class MongoContainer(dockerImageName: String) extends GenericContainer[MongoContainer](dockerImageName)
-
-  def getMongoContainer(mongoImageName: String = "mongo"): MongoContainer =
-    new MongoContainer(mongoImageName)
-      .withExposedPorts(mongoPort)
-      .withCommand("--replSet", "docker-rs")
-      .waitingFor(Wait.forLogMessage(".*Waiting for connections.*", 1))
-
-  def getConfigForMongoContainer(container: GenericContainer[_]): Config =
-    config("database.mongo.port" -> container.getMappedPort(mongoPort))
+  def getMongoContainer(mongoImageName: String = "mongo"): MongoDBContainer =
+    MongoDBContainer(DockerImageName.parse(mongoImageName))
```

Comment:
Where are you using this method ?

Reply:
It's obsolete. It comes from the previous solution. I'll fix it